DESCRIPTION

```diff
@@ -1,16 +1,23 @@
 Package: chipseq
 Title: chipseq: A package for analyzing chipseq data
 Version: 1.49.0
-Author: Deepayan Sarkar, Robert Gentleman, Michael Lawrence, Zizhen Yao
+Authors@R: c(
+   person("Deepayan", "Sarkar", role = "aut"),
+   person("Robert", "Gentleman", role = "aut"),
+   person("Michael", "Lawrence", role = "aut"),
+   person("Zizhen", "Yao", role = "aut"),
+   person("Oluwabukola", "Bamigbade", role = "ctb",
+   comment = "Converted Chipseq vignette from ChipseqFiles.Rnw to RMarkdown / HTML."),
+   person("Bioconductor Package Maintainer",
+        email = "maintainer@bioconductor.org", role = "cre"))
```

Comment:
Please use the same indent as the rest of the file, which looks like 8 spaces/tab. Note that the second line of the person entry should also be indented more; for example

    Authors@R: c(
            person("Bioconductor Package Maintainer",
                    email = "maintainer@bioconductor.org", role = "cre"))